src/main/java/org/interledger/cryptoconditions/Fulfillment.java

```diff
@@ -8,6 +8,13 @@
 
   Condition getCondition();
 
-  boolean verify(Condition condition, byte[] message);
+  /*
+   *  TODO:(0) change signature
+   *  verify(Condition condition, byte[] message) -> verify(byte[] message)
+   *  (condition must be "fetched" from the internal getCondition
+   *  in a similar way to the JS API)
+   */
+
+  boolean verify(Condition condition, byte[] message); // TODO:(?) In JS code is called validate. Rename?
 
```

Comment:
I don't think we can do that. Part of the validation process is to ensure that the Condition that is extracted from the Fulfillment matches another Condition.

E.g. Given C (from transfer request) and F (from commit notification). Derive Cf from F and ensure C == Cf AND F is valid.